iset.mm

```diff
@@ -151091,6 +151259,223 @@ the additive structure must be abelian (see ~ ringcom ), care must be
   $}
 
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Local rings
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  $c LRing $.
+  $( Extend class notation with class of all local rings. $)
+  clring $a class LRing $.
+
+  ${
+    $d x y r $.
+    $( A local ring is a non-zero ring where any sum which equals one has one
+       of the two summands which has a multiplicative inverse.  Any field is a
```

Comment:
"...where for any two elements summing to one, at least one is invertible." sounds less convoluted. Or even: "where anytime two elements sum to one, at least one is invertible".

Reply:
The first one looks better to me, I'll use that one.

Reply:
Done.